tensorflow/contrib/tensorrt/kernels/trt_engine_op.h

```diff
@@ -35,7 +35,7 @@ limitations under the License.
 
 namespace tensorflow {
 namespace tensorrt {
-class TRTInt8Calibrator;
+struct TRTInt8Calibrator;
```

Comment:
why `struct`?

Reply:
`TRTInt8Calibrator` is defined as a struct so this corrects it. This comes from the first commit of this PR.

Reply:
I see. I think class would be a better choice for `TRTInt8Calibrator` but that's separate from this PR.